src/cron/send_site_issues_notifications.js

```diff
@@ -0,0 +1,72 @@
+const { Sequelize, Op } = require('sequelize');
+const log = require('debug')('app:cron');
+const config = require('config');
+const Notifications = require('../notifications');
+const db = require('../db');
+const UseLock = require('../lib/use-lock');
+const sitesWithIssues = require('../services/sites-with-issues');
+
+// Purpose
+// -------
+// Send emails to projectmanagers just before the enddate of their project is reached
+// 
+// Runs every day
+module.exports = {
+  // cronTime: '*/10 * * * * *',
+  // runOnInit: true,
+  cronTime: '0 15 4 * * *',
+  runOnInit: false,
+  onTick: UseLock.createLockedExecutable({
+    name: 'send-site-issues-notifications',
+    task: async (next) => {
+
+      try {
+
+        let notoficationsToBeSent = {};
```

Comment:
Typo, should be changed throughout the file

```suggestion
        let notificationsToBeSent = {};
```

Reply:
Fixed